packages/js/test/plugins/auctionHouseModule/createBid.test.ts

```diff
@@ -333,7 +333,7 @@ test('[auctionHouseModule] it throws an error if Auctioneer Authority is not pro
   // Given we have an NFT.
   const mx = await metaplex();
   const seller = await createWallet(mx);
-  const nft = await createNft(mx);
+  const nft = await createNft(mx, {}, { owner: seller.publicKey });
```

Comment:
Side note: I've been changing a few things on the NFT module lately in a PR that aims to improve the DevEx and add support for fungible assets. Not too many things will change for Auction House but my PR has to change a few things there to update the interactions between the two modules. TL;DR; We might have a few conflicts to resolve soon. 😬